subiquity/server/controllers/tests/test_filesystem.py

```diff
@@ -497,6 +500,45 @@ async def test_lvm_20G_bad_offset(self, disk_size):
                 disk_size - (1 << 20), parts[-1].offset + parts[-1].size,
                 disk_size)
 
+    async def _sizing_setup(self, bootloader, ptable, disk_size, policy):
+        self._setup(bootloader, ptable, size=disk_size)
+
+        resp = await self.fsc.v2_guided_GET()
+        reformat = [target for target in resp.possible
+                    if isinstance(target, GuidedStorageTargetReformat)][0]
+        data = GuidedChoiceV2(target=reformat,
+                              capability=GuidedCapability.LVM,
+                              sizing_policy=policy)
+        await self.fsc.v2_guided_POST(data=data)
+        resp = await self.fsc.GET()
+
+        [vg] = child_matches(resp.config, type='lvm_volgroup')
+        [part_id] = vg['devices']
+        [part] = child_matches(resp.config, id=part_id)
+        part_size = part['size']  # already an int
+        [lvm_partition] = child_matches(resp.config, type='lvm_partition')
+        size = dehumanize_size(lvm_partition['size'])
+        return size, part_size
+
+    @parameterized.expand(bootloaders_and_ptables)
+    async def test_scaled_disk(self, bootloader, ptable):
+        size, part_size = await self._sizing_setup(
+                bootloader, ptable, 50 << 30, SizingPolicy.SCALED_FREE_SPACE)
+        # expected to be about half, differing by boot and ptable types
+        self.assertLess(20 << 30, size)
+        self.assertLess(size, 30 << 30)
```

Comment:
I don't know if my suggestion is better, so I'll leave that up to you:

```python
self.assertTrue((20 << 30) < size < (30 << 30))
```

Reply:
I started with that form but changed to the above, as the result is rather unhelpful if it assertTrue form fails:
```
subiquity/server/controllers/tests/test_filesystem.py:529: in test_scaled_disk
    self.assertTrue((20 << 30) < size < (30 << 30))
E   AssertionError: False is not true
```
In this form we get a size number
```
subiquity/server/controllers/tests/test_filesystem.py:530: in test_scaled_disk
    self.assertLess(20 << 30, size)
E   AssertionError: 21474836480 not less than -1
```